cmd/crd-replicator/main.go

```diff
@@ -48,7 +51,16 @@ func main() {
 		klog.Errorf("namespace env variable not set, please set it in manifest file of the operator")
 		os.Exit(-1)
 	}
-	clusterID, err := util.GetClusterID(k8sClient, clusterIDConfMap, namespaceName)
+
+	// 7 attempts with 30 seconds sleep between one another
+	// for a total of 3 minutes
+	backoff := wait.Backoff{
+		Steps:    3,
+		Duration: 15 * time.Second,
```

Comment:
The comment says 30 seconds but the backoff has 15 seconds. I would use 30 secs

Reply:
yes, copy&paste mistake I will fix